modules/c3-assisted-installer-preparing-image-generating.adoc

```diff
@@ -4,9 +4,13 @@
 
 :_mod-docs-content-type: PROCEDURE
 [id="c3-assisted-installer-preparing-image-generating_{context}"]
-= Generating the image in the {ai-full}
+= Generating the image in the Red Hat {ai-full}
 
-Create a cluster and download the discovery ISO image.
+Create a cluster and generate the {product-title} Discovery ISO image in the {ai-full} on the Red{nbsp}Hat portal. You can perform this step on your laptop, rather than on the bastion server or within environments such as Oracle Solution Center.
```

Comment:
If you do replace laptop in the other module, also change it here (and anywhere else it's used)